addons/medical_damage/initSettings.sqf

```diff
@@ -34,6 +34,15 @@
     true
 ] call CBA_fnc_addSetting;
 
+[
+    QEGVAR(medical,const_painUnconscious),
```

Comment:
Rename this? Seems kinda weird to have a setting with the prefix `const`

Reply:
what if someone use it?
Or using undocumented variables is "at one's own risk"?